config/settings.yml

```diff
@@ -1439,19 +1439,6 @@ vanotify:
   status_callback:
     bearer_token: fake_bearer_token
 
-# Settings to connect to the drupal forms graphql api
-va_forms:
-  drupal_username: ~
-  drupal_password: ~
-  drupal_url: https://fake-url.com
-  form_reloader:
-    enabled: false
-  # A notification for when a forms url has changed
-  slack:
-    enabled: false
-    api_key: ""
-    channel_id: ""
```

Comment:
A PR will be opened in the manifests repo to remove these settings there as well. After that is done, we will also make a request to the platform team to remove the secrets.